hw/ip/pattgen/doc/checklist.md

```diff
@@ -163,18 +163,18 @@ Review        | [V2_CHECKLIST_SCOPED][]               | Done        |
  Type         | Item                                    | Resolution  | Note/Collaterals
 --------------|-----------------------------------------|-------------|------------------
 Documentation | [DESIGN_DELTAS_CAPTURED_V2][]           | N/A         |
-Documentation | [DV_DOC_COMPLETED][]                    | Not Started |
+Documentation | [DV_DOC_COMPLETED][]                    | Done        |
 Testbench     | [FUNCTIONAL_COVERAGE_IMPLEMENTED][]     | Not Started |
 Testbench     | [ALL_INTERFACES_EXERCISED][]            | Done        |
 Testbench     | [ALL_ASSERTION_CHECKS_ADDED][]          | Done        |
 Testbench     | [SIM_TB_ENV_COMPLETED][]                | Done        |
 Tests         | [SIM_ALL_TESTS_PASSING][]               | Done        |
```

Comment:
PLease see: https://reports.opentitan.org/hw/ip/pattgen/dv/latest/results.html
There are several failing tests. Please triage and get all V1 and V2 tests passing before marking this as done.

Reply:
@sriyerg it has been fixed by Cindy. In addition, the alert test was added in PR #7015